book/object/ownership.md

```diff
@@ -21,7 +21,37 @@ to others for modification or transfer. This level of ownership clarity is a sig
 over other blockchain systems, where ownership definitions can be more ambiguous, and smart
 contracts may have the ability to alter or transfer assets without the owner's consent.
 
-<!-- TODO: add an example -->
+Let's illustrate this with a simple example:
+
+A `Key` structure will be minted and transferred to the publisher at the same time as the smart contract
+is published. Only the owner of this `Key` has the right to perform subsequent operations.
+
+```move
```

Comment:
How could we simplify this example so it maybe doesn't feature code at all? Looking at other sections on this page, and they're not as verbose.